src/components/Card/Card.js

```diff
@@ -1,16 +1,11 @@
 import React from "react";
 import "./Card.css";
 
-const Card = ({ name, race, gender, realm, birthday, wikiUrl, id }) => {
+const Card = ({ dialog, key, id }) => {
   return (
-    <div className="character-card" id={id}>
-      <h3>{name}</h3>
-      <p>{race}</p>
-      <p>{gender}</p>
-      <p>{realm}</p>
-      <p>{birthday}</p>
-      <p>{wikiUrl}</p>
-      <p>{id}</p>
+    <div className="quote-card" key={key}>
```

Comment:
I would suggest maybe setting the key for each card in the List component when you're mapping through the data